docx/header.py

```diff
@@ -7,13 +7,13 @@
 from __future__ import (
     absolute_import, division, print_function, unicode_literals
 )
-
 from .shared import ElementProxy
 
 
 class Header(ElementProxy):
     """
-    The default page header of a section.
+    Proxy for ``<w:hdr>`` element in this section, having primarily a
+    container role.
```

Comment:
This commit is merged now, these comments are just for reference for future commits :)
https://github.com/python-openxml/python-docx/commits/feature/header

I left this change out. The docstring is for the library end-user (developer using python-docx) because it appears in the API documentation. So the details of what's being proxied wouldn't generally be a concern for them. I know I've used the "container role" phrase elsewhere, but I'm thinking if we want to elaborate we'd want to do so with specifics that would be helpful to a developer learning how to use this object. I'm sure we'll find more tidbits to add here as we go.